content/features/featuresDeepDive/postProcesses/IBLShadowsRenderingPipeline.md

```diff
@@ -0,0 +1,195 @@
+1. If `triPlanarVoxelization` is enabled (the default), the voxelization will be done three times, once along each axis. The purpose of this is to avoid missing triangles that are parallel to the camera's line-of-sight. Disabling `triPlanarVoxelization` will speed up voxelization time but can often result in missing geometry.
+1. After the voxel grid is generated, the heirarchical mips need to be generated for it. These are successively smaller and smaller textures that are used by the voxel tracing algorithm to efficiently traverse the voxel grid and find ray-geometry intersections, something like an octree.
+
+### Each Frame
+
+Once the CDF maps and voxel grid are created, there are several fullscreen passes that happen every frame to render the shadows. These all happen before your scene is rendered with full materials and lighting.
+
+1. The [Geometry Buffer Renderer](https://doc.babylonjs.com/typedoc/classes/babylon.geometrybufferrenderer) renders the entire scene into several render target textures needed by the shadow pipeline. These include world-space normals, screen-space depth, world-space positions and motion vectors. These targets may also be shared by other pipelines that need them.
+1. A "Voxel-tracing" pass will use the CDF maps and voxel grid to produce a shadow value for every pixel.
+   1. Several samples of the IBL can be taken (controlled using the `sampleDirections` property). For each sample:
+      1. A light direction is generated using the CDF maps. The direction is randomly selected based on the intensity of the light in that direction. i.e. the probability that a particular direction will be chosen is relative to the brightness of the IBL in that direction.
+      1. The voxel grid is traversed in this direction to see if it will intersect any geometry. The mips of the voxel grid are key to traversing the grid efficiently.
+      1. A screen-space shadow sample (see [below](https://doc.babylonjs.com/features/featuresDeepDive/postProcesses/IBLShadowsRenderingPipeline#screen-space-shadows)) is also taken using this light direction and is combined with the voxel sample.
+      1. Two shadow values are then generated. The first is the simple accumulation of the shadow amount from each sample. The second is scaled by a view-dependent factor to approximate shadowing of specular lighting from the sampled direction.  
+   1. The total shadow contribution of both diffuse shadowing and specular shadowing is divided by the number of samples to get the amount of raw shadowing for this frame. Taking more samples per frame will decrease the noise in the image.
+1. A blur pass then blurs the shadows to help remove some of the noise.
+1. An accumulation pass will then combine the new shadow frame with shadows generated in previous frames. This has the effect of building up a smooth image of the shadows over several frames. The speed at which this happen can be adjusted using the `shadowRemenance` property.

```

Comment:
After checking, it seems that “remenance” is spelled “remanance”! I don't think it's a problem to update the name of the property, as the feature will be official only in 8.0.